Latest SDK for Android

SailingSteve opened the pull request:
Updated cordova-plugin-facebook4, cordova-ios, and cordova-android to the latest.
Some documentation updates.

SailingSteve committed:
Latest SDK for Android
Updated cordova-plugin-facebook4, cordova-ios, and cordova-android to the latest.
Some documentation updates.

DaleMcGrew commented:
Hi @SailingSteve, it looks like there are conflicts in 4 of the files you checked in. Could you please take a look?
platforms/android/android.iml
platforms/android/android.json
platforms/android/app/src/main/res/xml/config.xml
platforms/android/build.gradle

SailingSteve committed:
Latest SDK for Android
Updated cordova-plugin-facebook4, cordova-ios, and cordova-android to the latest.
Some documentation updates.

SailingSteve committed:
Latest SDK for Android
Updated cordova-plugin-facebook4, cordova-ios, and cordova-android to the latest.
Some documentation updates.

SailingSteve committed:
Latest SDK for Android
Updated cordova-plugin-facebook4, cordova-ios, and cordova-android to the latest.
Some documentation updates.
